versioned_docs/version-5.x/headers.md

```diff
@@ -44,7 +44,7 @@ function StackScreen() {
       <Stack.Screen
         name="Profile"
         component={ProfileScreen}
-        options={({ route }) => ({ title: route.params.name })}
```

Comment:
Hey @maksshilov, thanks for your PR! 

this is the expected behavior for this example, we want to show how to change the `title` based on `params` prop.